crates/compiler/test_syntax/tests/snapshots/pass/result_try_question_multiple_defs.moduledefs.formatted.roc

```diff
@@ -0,0 +1,4 @@
+main =
+    a? "Bar"
+x = B.b? "Foo"
+c? x
```

Comment:
This is not correct, all statements should be parsed as part of the `main`
https://github.com/roc-lang/roc/pull/6844#discussion_r1662959786